mantidproject/mantid
# New compiler for release notes

**Description of work.**
This PR covers most of the work needed to automatically compile seperate release notes during sprints. It does this by using the new amalgamate sphinx directive. As this pulls direct from the necessary folders the placeholder files for compiling in a previous iteration of the project design can be removed.

**To test:**

- Open `/docs/source/release/v6.4.0/diffraction.rst`
- Add the following script to the bottom of the page
```
Testing
#######
Engineering Diffraction
-----------------------

New features
############
.. amalgamate:: /Diffraction/Engineering/New_features

```
- Go to the folder `/docs/source/release/v6.4.0/Diffraction/Engineering/New_features`
- Add two or more .rst files containing text (as if they were two new separate release notes
- delete your` build/docs` folder
- run CMake and rebuild the docs
- open `build/docs/html/release/v6.4.0/diffraction.html`
- At the bottom of the page should be the new text you added to the bottom of `diffraction,rst` and the text from the two (or more) files you created under the New Features heading.

Part of #33593 


*This does not require release notes* because **it should not affect users**

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
